python_modules/libraries/dagster-azure/dagster_azure/blob/compute_log_manager.py

```diff
@@ -279,6 +279,20 @@ def download_url_for_type(self, log_key: Sequence[str], io_type: ComputeIOType):
         self._download_urls[blob_key] = url
         return url
 
+    def get_log_manager_metadata(self) -> dict[str, str]:
+        metadata = super().get_log_manager_metadata()
+        return {
+            **metadata,
+            "storage_account": self._storage_account,
+            "container": self._container,
```

Comment:
Trying to think through whether it's at all concerning to store container name and storage account name... 

I think no, since this information is actually already encoded in the url.

Reply:
yes, it's already revealed in the pre-existing url.

generally: it's not a zero concern, but a minimal one.

It doesn't give any ability to auth, but it's still information that is leaked and could be useful in aggregate with more information and exploit by an attacker, generally speaking.

ex: knowing that this particular entity exists, and maybe the name of that entity can allow a malicious actor to get more intelligence, ie if the storage or container's name contain information that identifies type of infrastructure, a project or customer name or any other information.

However, here in this case I think it's reasonably justifiable.